README.md

```diff
@@ -1,13 +1,104 @@
-erl_optics
-=====
+# erl_optics
 
-An OTP library
+An OTP application for [optics](https://github.com/RAttab/optics).
 
-### Requirements
+## Requirements
+-----
 
 * Erlang OTP >= 21
+* [optics](https://github.com/RAttab/optics) v1.1.9
 
-Build
+## Building
 -----
-
     $ rebar3 compile
+
+## Usage
+-----
+
+For now, the only working mode is Carbon.
+
+need env :
+```
+  {erl_optics, [
+    {hostname, "<address of carbon host>"},
```

Comment:
I think it's better to have well-formatted yet obvious examples here, e.g `https://my-carbon-host.com`